package.json

```diff
@@ -82,14 +88,15 @@
     "react-dom": "^17.0.1",
     "react-redux": "^7.2.1",
     "react-router-dom": "^5.2.0",
-    "redux": "^4.0.0"
-  },
-  "dependencies": {
+    "react-test-renderer": "17.0.1",
+    "redux": "4.0.5",
     "@fortawesome/free-solid-svg-icons": "^5.8.1",
     "@fortawesome/react-fontawesome": "^0.1.4",
+    "algoliasearch": "^4.8.5",
```

Comment:
I lost track of slack, I will try to dig it up there was an issue when I had introduced some peer deps, in build process of some other MFEs that use this project (I was surprised to learn than even ecommerce builds somehow depend on this, let me check and get back to you on this) @adamstankiewicz may also be able to add context

Reply:
slack thread for relevant convos: https://edx-internal.slack.com/archives/CM4RPEYEQ/p1606157355298600

Reply:
frontend-enterprise is used by [frontend-component-header-edx](https://github.com/edx/frontend-component-header-edx/blob/55634492ac2b300467d39d0c6ed11be4b0760c7b/src/Header.jsx#L13) to fetch whether a learner is linked to an Enterprise with the learner portal enabled, and if so changes the logo image/link and the avatar dropdown menu items to point to https://enterprise.edx.org/{slug} instead of the b2c links.

This header component is used by various MFEs, hence the dependency by some of our apps. Similar story with the frontend-enterprise dependency in frontend-app-learning, which has its own header.

I'm not recalling the particular issue @binodpant is referring to, though, unfortunately.

Reply:
We're making this a major version bump, so it won't get automatically pulled in, and people can add the dependencies need to satisfy the peerDependencies.